explainability-service/src/test/java/org/kie/trustyai/service/endpoints/metrics/UniversalListingEndpointTest.java

```diff
@@ -41,13 +42,17 @@ class UniversalListingEndpointTest {
     @BeforeEach
     void populateStorage() throws JsonProcessingException {
         storage.get().emptyStorage();
-        scheduler.get().getDirRequests().clear();
-        scheduler.get().getSpdRequests().clear();
+        scheduler.get().getAllRequests().clear();
         final Dataframe dataframe = datasource.get().generateRandomDataframe(1000);
         datasource.get().saveDataframe(dataframe, MODEL_ID);
         datasource.get().saveMetadata(datasource.get().createMetadata(dataframe), MODEL_ID);
     }
 
+    @AfterEach
+    void clean() {
+        scheduler.get().getAllRequests().clear();
```

Comment:
You do it already in the `@BeforeEach` so it should not be necessary.

Reply:
Not neccesarily, right? This ensures that the last test also cleans up, before moving onto the next test file?

Reply:
I think there is also the `@AfterAll` hook but your comment made me think: I'm not totally sure if Quarkus already does a new initialisation for every test file or not, I guess so to be honest, so it should not be possible to "share" data between two different tests in this way.
Btw it is not a big deal, it is useful to check this because now the cleanup is quite simple but if the object are not initialized again for every test the risk of unexpected behaviour is quite high